### PR TITLE
Added utility to invoke DeleteResourceFromGraphite flow from monitoring-integration

### PR DESCRIPTION
**Pull request by shtripat**
tendrl-bug-id: Tendrl/gluster-integration#437
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
@r0h4n @GowthamShanmugam @rishubhjain plz review

File: tendrl/commons/utils/monitoring_utils.py [hunk 1]
```diff
@@ -26,3 +26,28 @@ def update_dashboard(res_name, res_type, integration_id, action):
...
+    _params = {
+        "TendrlContext.integration_id": NS.tendrl_context.integration_id,
+        "TendrlContext.cluster_name": NS.tendrl_context.cluster_name,
+        "Trigger.resource_name": res_name,
+        "Trigger.resource_type": res_type,
+        "Trigger.action": action
+    }
+    _job_payload = {
+        "tags": ["tendrl/integration/monitoring"],
+        "run": "monitoring.flows.UpdateGraphite",
```
Comment (r0h4n) [inline tendrl/commons/utils/monitoring_utils.py]: Change flow name to 

monitoring.flows.DeleteResourceFromGraphite

https://github.com/Tendrl/monitoring-integration/pull/154/files#r142021865

File: tendrl/commons/utils/monitoring_utils.py [hunk 2]
```diff
@@ -26,3 +26,28 @@ def update_dashboard(res_name, res_type, integration_id, action):
     ).save()
 
     return _job_id
+
+
+def update_graphite(res_name, res_type, integration_id, action):
```
Comment (r0h4n) [inline tendrl/commons/utils/monitoring_utils.py]: delete_resource_from_graphite instead of update_graphite

Comment (shtripat) [reply tendrl/commons/utils/monitoring_utils.py]: Done

File: tendrl/commons/utils/monitoring_utils.py (hunk 1 above)
Comment (shtripat) [reply tendrl/commons/utils/monitoring_utils.py]: Done

**Pull request merged**